### ISSUE TITLE
Implement basic view of new events

### ISSUE THREAD

**Issue by aptmac** [rh-jmc-team/jmc-agent-plugin#7]
After loading an XML, it'd be nice to have a visual representation of what changed (e.g., new events).

**Comment by jessyec-s** [comment]
This is blocked by [JMC-6715](https://bugs.openjdk.java.net/browse/JMC-6715) as currently there is no way to receive information from the agent that identifies the transforms that are currently instrumented.

EDIT this feature has now been implemented in the agent
